File: src/main/java/com/morak/performancetracker/configuration/DescriptorConfigration.java
package com.morak.performancetracker.configuration;

import com.fasterxml.jackson.databind.ObjectMapper;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class DescriptorConfigration {

    @Bean
    public ObjectMapper objectMapper() {
        return new ObjectMapper();
    }
}

File: src/main/kotlin/com/morak/performancetracker/configuration/DescriptorConfigration.kt
package com.morak.performancetracker.configuration

import com.fasterxml.jackson.databind.ObjectMapper
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class DescriptorConfigration {
    @Bean
    fun objectMapper(): ObjectMapper {
        return ObjectMapper()
    }
}
